deploy.py

```diff
@@ -921,6 +921,13 @@ def main(args):
       const="console_test",
       help=("Compiles and installs the console_test example that tests the "
             "console driver with messages of various lengths."))
+  apps_group.add_argument(
+      "--nfct_test",
```

Comment:
It seems that the `examples/nfct_test.rs` file was added and then deleted as part of this PR.
Therefore running `deploy.py --nfct_test` will result in an error.